cmd/server/stdio_handshake_test.go

```
@@ -0,0 +1,103 @@
+}
+
+func buildServerBinary(t *testing.T) string {
+	t.Helper()
+
+	binPath := filepath.Join(t.TempDir(), "luno-mcp-test")
+
+	wd, err := os.Getwd()
+	require.NoError(t, err)
+	repoRoot := filepath.Clean(filepath.Join(wd, "..", ".."))
+
+	cmd := exec.Command("go", "build", "-o", binPath, "./cmd/server")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)